utils/split_model_tests.py

```diff
@@ -0,0 +1,22 @@
+import os
```

Comment:
Would be nice to add a small comment as to what this file does, where it's used from, + copyright

Reply:
yes, will update this file!